src/10-bf.conf

```diff
@@ -1,4 +1,4 @@
 [Service] 
 Environment="KUBELET_CONFIG_ARGS=--config=/var/lib/kubelet/config.yaml" 
 ExecStart= 
-ExecStart=/usr/bin/kubelet $KUBELET_CONFIG_ARGS --container-runtime=remote --container-runtime-endpoint=unix:///run/containerd/containerd.sock
+ExecStart=/usr/bin/kubelet $KUBELET_CONFIG_ARGS
```

Comment:
Can you explain what was the error message you show with the old config? 
Is the --container-runtime --container-runtime-endpoint should move to the config.yaml?

Reply:
```
Mar 20 07:05:22 cloud-dev-10-bf kubelet[167751]: E0320 07:05:22.130808  167751 run.go:74] "command failed" err="failed to parse kubelet flag: unknown flag: --container-runtime"
```
```
Mar 20 07:08:13 cloud-dev-10-bf kubelet[168699]: Flag --container-runtime-endpoint has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
```